tests/io/filereaders/TestXMLReader.cpp

```diff
@@ -0,0 +1,78 @@
+#include <gtest/gtest.h>
+
+#include "data/FileLoader.h"
+#include "io/input/xml/XMLFileReader.h"
+#include "particles/ParticleContainer.h"
+#include "utils/ArrayUtils.h"
+
+#define EXPECT_CONTAINS_POS_NEAR(list, point, tol) \
+    EXPECT_TRUE(std::find_if(list.begin(), list.end(), [&](auto& x) { return ArrayUtils::L2Norm(x - point) < tol; }) != list.end());
+
+/*
+ * Test if a XMLFileReader reads the correct data out of the files.
+ */
```

Comment:
What does this commented code do? is it still relevant?

Reply:
Ill removed it